packages/itinerary-body/src/TransitLegBody/index.tsx

```diff
@@ -210,18 +210,13 @@ class TransitLegBody extends Component<Props, State> {
           )}
 
           {/* The Alerts body, if visible */}
-          <VelocityTransitionGroup
-            enter={{ animation: "slideDown" }}
-            leave={{ animation: "slideUp" }}
-          >
-            {expandAlerts && (
-              <AlertsBody
-                alerts={leg.alerts}
-                AlertIcon={AlertBodyIcon}
-                timeZone={timeZone}
-              />
-            )}
-          </VelocityTransitionGroup>
+          <AnimateHeight duration={500} height={expandAlerts ? "auto" : 0}>
+            <AlertsBody
```

Comment:
Previously, `<AlertsBody>` was rendered conditionally. If rendering it unconditionally going forward, you'll need to check in that component's render function for props that could be null, such as `alerts` where some of the snapshot tests are failing.